content/klc/S2.1.adoc

```diff
@@ -2,14 +2,15 @@
 brief = "General symbol naming guidelines"
 +++
 
-. Library naming should not be duplicated in footprint name
+. Library name should not be duplicated in symbol name
 . If symbol with same name exists for multiple manufacturers, the manufacturer name is written first
 . Specific manufacturer name (for atomic parts)
 . Type of symbol (for generic parts)
 .. May be shortened for common components (e.g. `Conn` for `Connector`)
 .. Reference designator may be substituted for common components (e.g. `D`, `C`, `LED`)
-. Part name should include extension for specific footprint if required (e.g. `SOIC`)
+. Part name should include extension for specific footprint if the same part is produced by multiple manufacturers that have conflicting package name suffixes (e.g. `SOIC`)
```

Comment:
Read in isolation, yes, perhaps. All examples given in this font style are the symbol name piece that is part of the full symbol name, and this is the same.

Reply:
That is true, but perhaps the wording can be improved to reduce the confusion (though I can't come up with something quickly). Maybe say "(e.g. add `SOIC`) to disambiguate?

Reply:
What if underscores are added before each example? So changing `SOIC` to `_SOIC` and elsewhere there are similar examples.

Reply:
Adding the underscore would improve things a little, though not really help to prevent misreading his in the way I originally suggested (since a MFG "part name suffix" might also use an underscore). Often, these MFG suffixes do not use underscores, so it does help a little.

Regardless, adding the underscore here (and to `_SOIC8` on the next line) would be good for consistency (since it is also present for `_Split` further down. The other examples, e.g. `Q_NPN_BCE` are full symbol names, so the underscore is present inside the name).